.github/workflows/tests.yml

```diff
@@ -653,3 +653,49 @@ jobs:
 
     - name: Exercise RGW again
       run: ~/actionutils.sh testrgw "newFile"
+
+  replication-tests:
+    name: Test MicroCeph Cluster features.
```

Comment:
This should probably read "Test MicroCeph replication features" or similar

Reply:
ack